<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event</class>
 <widget class="QDialog" name="Event">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="main_groupbox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="duration_spinbox">
          <property name="suffix">
           <string> h</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QDoubleSpinBox" name="f_length_dspinbox">
          <property name="suffix">
           <string> km</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Anzahl Fahrer:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Renndauer:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="drivers_spinbox">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Rennbeginn:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Formations-Rundenlänge:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QDoubleSpinBox" name="length_dspinbox">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string> km</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Rundenlänge:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTimeEdit" name="start_timeedit">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>12</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" rowspan="2">
          <widget class="QGroupBox" name="drivers_groupbox">
           <property name="title">
            <string>Fahrer</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="name_lineedit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Erwartete Rundenzeit:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="expected_duration_spinbox">
             <property name="suffix">
              <string> min</string>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Anzahl Runden am Stück:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="interval_spinbox">
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Glympse ID:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="glympse_lineedit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPlainTextEdit" name="editor"/>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <widget class="QPushButton" name="reset_pushbutton">
       <property name="text">
        <string>Reset Editor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="delete_pushbutton">
       <property name="text">
        <string>ALLE Aufzeichnungen  löschen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Event</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Event</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
